bittensor/_neuron/text/advanced_server/sample_config.txt

```diff
@@ -0,0 +1,54 @@
+axon.backward_timeout: 20
+axon.forward_timeout: 10
+axon.ip: '[::]'
+axon.max_workers: 10
+axon.maximum_concurrent_rpcs: 400
+axon.port: 8091
+axon.priority.max_workers: 10
+axon.priority.maxsize: -1
+
+dataset.batch_size: 10
+dataset.block_size: 20
+dataset.data_dir: ~/.bittensor/data/
+dataset.dataset_name: train
+dataset.max_corpus_size: 10000.0
+dataset.num_workers: 0
+dataset.save_dataset: false
+
+logging.debug: false
+logging.logging_dir: ~/.bittensor/miners
+logging.record_log: false
+logging.trace: false
+
+server:
```

Comment:
I think these are the old settings; we made a few changes to the defaults and added a few settings, checkout the Akatsuki branch for details

Reply:
all done! should neuron.full_path be in the config tho?

Reply:
Yea, lets remove it, because it is currently unimplemented.